tests/integration_testnet/api/http/delegates.test.js

```diff
@@ -35,7 +35,7 @@ const delegateSchema = {
 	secondPublicKey: 'string',
 	username: 'string',
 	vote: 'string',
-	status: 'string',
+	// status: 'string', uncomment when testnet updated to SDKv4
```

Comment:
`status` is now a mandatory property. Kindly enable it in the schema.

```suggestion
	status: 'string',
```

Reply:
done!